src/utils/hwManager.js

```diff
@@ -22,10 +25,25 @@ const getAccountsFromDevice = async () => {
  * signSendTransaction - Function.
  * This function is used for sign a send transaction.
  */
-const signSendTransaction = () => {
-  // TODO implement logic for this function
-  signTransaction();
-  throw new Error('not umplemented');
+const signSendTransaction = async (account, data, pin = null) => {
+  const transactionObject = createSendTX(
+    account.info.LSK.publicKey,
+    data.recipientId,
+    data.amount,
+    data.data,
+  );
+
+  const index = (typeof pin === 'string' && pin !== '')
+    ? calculateSecondPassphraseIndex(account.hwInfo.derivationIndex, pin)
```

Comment:
We don't support pin and second passphrase for hardware wallets. 

More details about it here:
https://github.com/LiskHQ/lisk-hub/issues/1101#issuecomment-408371015
https://github.com/LiskHQ/lisk-hub/issues/1101#issuecomment-418631999
https://github.com/LiskHQ/lisk-hub/issues/1101#issuecomment-418651258

Reply:
thx this one is fixed